src/main/java/owen/Ui.java

```diff
@@ -0,0 +1,34 @@
+package owen;
+
+import java.util.Scanner;
+
+/**
+ * Terminal UI handling user input and output.
+ */
+public class Ui {
+    private final Scanner scanner;
+
+    /**
+     * Create new Terminal UI for handling user input and output.
+     */
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Output a formatted response to standard output.
```

Comment:
Method header comments should start in the form "Outputs a ..."
https://se-education.org/guides/conventions/java/intermediate.html#comments